src/ServiceManagement/Compute/Compute.Tests/Compute.Tests.csproj

```diff
@@ -36,6 +36,7 @@
     <None Include="OneWebOneWorker.cspkg">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="packages.config" />
```

Comment:
I don't think you need these changes. Probably generated by VS or the build process. Please try to revert them.